modules/swagger-codegen/src/main/resources/python/model.mustache

```diff
@@ -6,10 +6,6 @@ import pprint
 import re  # noqa: F401
 
 import six
-{{#imports}}{{#-first}}
```

Comment:
What about the scenario where the discriminator is used?
The lack of import will result in invalid code due to the missing import.

Reply:
Does this discriminator work if modules can't be imported due to circular dependencies ? 

Maybe there is another way to provide paths/data for the discriminator - I don't know, I don't use it. Such imports make more problems for standard usage of generated code.

Reply:
What is the status of this PR? I have a similar fix and was about to submit it before stumbling into this PR. 
We are seeing models created with circular imports too which makes the generated code useless unless they are manually commented out!